### PR TITLE
Fix runtime log

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

**Comment by sebastianbenz** [comment]
Force merged to unbreak the build.

File: pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-html-layout/index.md [hunk 1]
```diff
@@ -271,7 +271,7 @@ The `fallback` attribute can be set on any HTML element, not just AMP elements.
 
 ### `noloading` <a name="noloading"></a>
 
-The `noloading` attribute indicates whether the "loading indicator" should be turned off for this element. Many AMP elements are allow-listed to show a "loading indicator", which is a basic animation that shows that the element has not yet fully loaded. The elements can opt out of this behavior by adding
+The `noloading` attribute indicates whether the "loading indicator" should be turned off for this element. Many AMP elements are white-listed to show a "loading indicator", which is a basic animation that shows that the element has not yet fully loaded. The elements can opt out of this behavior by adding
```
Comment (DerekNonGeneric) [inline pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-html-layout/index.md:274]: Is this change necessary?

Comment (sebastianbenz) [reply pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-html-layout/index.md:274]: This change has been auto imported from the ampproject/amphtml repo. Fixed it there: https://github.com/ampproject/amphtml/pull/28955

Comment (sebastianbenz) [reply pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-html-layout/index.md:274]: Thanks for spotting!
